microsoft/testsuites/dpdk/common.py

```diff
@@ -54,8 +54,7 @@ def install_required_packages(
         self, node: Node, extra_args: Union[List[str], None]
     ) -> None:
         os = node.os
-        if not isinstance(os, Posix):
-            return
+        assert isinstance(os, Posix)
```

Comment:
Add a failure message, so it can show in the test results error message.

Reply:
added

Reply:
Please add the actual type like other type assertion, so the error message contains what actual OS it is.

Reply:
added